NAMESPACE

```diff
@@ -9,6 +9,7 @@ export(as_listing)
 export(get_keycols)
 export(is_keycol)
 export(listing_dispcols)
+export(pag_lsting_indices)
```

Comment:
Lets go ahead and call it `pag_listing_indices`, since thats only one more letter to spell it out. After that, I think we're good here.